src/main/java/com/microsoft/graph/models/IProgressCallback.java

```diff
@@ -0,0 +1,12 @@
+package com.microsoft.graph.models;
+
+public interface IProgressCallback {
```

Comment:
We should drop that interface and use a BiConsumer instead to people can pass lambdas in https://docs.oracle.com/javase/8/docs/api/java/util/function/BiConsumer.html
Or at the very least tag this interface as functional